release/lightning_tests/compute_tpl.yaml

```diff
@@ -0,0 +1,22 @@
+cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
```

Comment:
Add a small comment at the top to describe what the compute is for this test? (N x node_type = X cpus, Y gpus total)

Reply:
sounds good